### ISSUE TITLE
The plugin needs an option to setup position

### ISSUE THREAD

**Issue by micobarac** [caroso1222/notyf#36]
The plugin needs an option to setup position (top-left, top-center, top-right, bottom-left, bottom-center, bottom-right, middle-left, middle-center, middle-right). 
Currently, it's stuck to the bottom right. 

Btw, it's a great plugin!
Thanks for all the effort you've put into it.

**Comment by caroso1222** [comment]
That's a great idea. I've been postponing this feature just for simplicity reasons, didn't know people needed it. I'll work on this soon!

**Comment by amusedberry** [comment]
At least for my use case, I only require a specific position.
This is not optimal, but a workaround while this feature is released is to overwrite the `.notyf` class:

```ts
// module.ts
import { Notyf } from 'notyf'
import 'notyf/notyf.min.css'
import '../utils/messages.css'
```

```css
/* messages.css */
.notyf {
  top: 60px;
  right: 20px;
  bottom: auto;
}
```

Notice how I set `bottom` to auto, to overwritte `bottom: 20px;` on the original file, and added a `top: 60px;` to position them to the top, after my navigation bar.

Same thing can be done with `right` if you want to position your toasts to the left.

**Comment by caroso1222** [comment]
Closing. This feature has been introduced in https://github.com/caroso1222/notyf/pull/48 and is available in `3.2.0`.

Thanks for the report and the follow-up!
